butler-bot/bot.js

```diff
@@ -63,4 +68,46 @@ client.on('message', message => {
     .then(() => { logger.info('Successfully added the role') })
 })
 
+client.setInterval((servers) => {
+  const itMuslimServer = servers.get(IT_MUSLIM_SERVER_ID)
+
+  const incompleteMembers = itMuslimServer.members.filter((member) => {
+    return !member.roles.has(FULL_MEMBER_ROLE_ID)
+  })
+  /* console.log(incompleteMembers) */
+
+  const incompleteMembersName = incompleteMembers.map((item) => {
```

Comment:
Take as a rule to put **s** at the end of your words. Let's say not car**s**Accident, but carAccident**s**.
```suggestion
  const restrictedMemberNames = incompleteMembers.map((item) => {
```